run_simfit_data_fullAngularMass_Swave.sh

```diff
@@ -7,8 +7,8 @@ nsam=${1}
 q2stat=${4}
 
 XGBv=8
-localFile=1
-fitopt=1
+localFile=0
+fitopt=0
```

Comment:
I believe we would like to keep localFile = 1, since we are now copying all the files locally
For fitopt, are the data-like Jpsi subsamples using exponential bkg mass shape?

Reply:
as you wish for localFile, I don't use it personally but not problem.
yes for the subsamples we used the exponential for ANv9, but we can change it even though I don't know how it behaves with small stat